poucave/app.py

```diff
@@ -19,6 +19,32 @@
 HTML_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), "html")
 
 
+class Checks:
+    def __init__(self, conf):
+        self.all = []
+        for project, checks in conf["checks"].items():
+            for name, params in checks.items():
+                check = Check(project, name, **params)
+                self.all.append(check)
+
+    def get(self, project=None, name=None):
```

Comment:
You proposed `lookup` in a video call, and I think that makes more sense (it fails if missing). But I think we should define another method for `get_all`.